apps/verticalhome/js/configurator.js

```diff
@@ -20,6 +20,8 @@
   var singleVariantApps = {};
   var simPresentOnFirstBoot = true;
 
+  var DEFAULT_COLUMNS = 2;
```

Comment:
I can change it to const, but this declaration like var  is consistent with existing code in this file